src/flamenco/runtime/program/fd_bpf_loader_v3_program.c

```diff
@@ -50,7 +50,7 @@ fd_bpf_loader_v3_is_executable( fd_exec_slot_ctx_t * slot_ctx,
   fd_bincode_decode_ctx_t ctx = {
     .data    = (uchar *)meta     + meta->hlen,
     .dataend = (char *) ctx.data + meta->dlen,
-    .valloc  = slot_ctx->valloc,
+    .valloc  = fd_scratch_virtual(),
```

Comment:
```suggestion
    .valloc  = instr_ctx->valloc,
```

Reply:
we don't have access to the instr context in this function